src/main/java/frc/robot/RobotContainer.java

```diff
@@ -76,6 +77,7 @@ public RobotContainer(RobotId robotId) {
       case TestBed2:
         intake = new Intake();
         index = new StorageIndex();
+        SmartDashboard.putData("IntakeNote", new IntakeNote(intake, index));
```

Comment:
why can the testbed2 intake a note?

Reply:
It has an intake, so why not add the intake note command for testing? We will change this if we plug something else into the test bed.